.github/ISSUE_TEMPLATE/4-report-a-flaky-test.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Report a flaky test
+about: Report a flaky test in our CI
+labels: "CI / flaky test"
+
+---
+
+<!--
+Thank you for reporting a flaky test.
+
+Flaky tests are tests that fail occaisonally in Node.js CI, but not consistently enough to block PRs from landing, 
+or that are failing in CI jobs or test modes that are do not run for every PR. 
+Since they are failing in CI, all the information for the template below should be available.
```

Comment:
We already ask for the information below and simplifying the text will hopefully help people actually reading the information. I therefore suggest to remove this sentence.

```suggestion
```